app/controllers/application_controller/timelines.rb

```diff
@@ -298,9 +298,9 @@ def tl_build_init_options(refresh = nil)
 
     if @tl_options[:tl_show] == "policy_timeline"
       @tl_options[:all_results] = {}
-      @tl_options[:all_results]["Both"] = "both"
-      @tl_options[:all_results]["True"] = "success"
-      @tl_options[:all_results]["False"] = "failure"
+      @tl_options[:all_results][_("Both")] = "both"
+      @tl_options[:all_results][_("True")] = "success"
+      @tl_options[:all_results][_("False")] = "failure"
```

Comment:
question why hash key should be I18N.  actually suggest they become symbols: `:both`, `:true`, `:false`